tests/phpunit/tests/theme/wpGetGlobalStylesheet.php

```diff
@@ -199,4 +210,64 @@ public function test_variables_in_classic_theme_with_presets_using_defaults() {
 		remove_theme_support( 'editor-font-sizes' );
 	}
 
+	/**
+	 * Tests that stored CSS is enqueued.
+	 *
+	 * @ticket 56467
+	 *
+	 * @covers ::wp_enqueue_stored_styles
+	 */
+	public function test_should_enqueue_stored_styles() {
+		$core_styles_to_enqueue = array(
+			array(
+				'selector'     => '.saruman',
+				'declarations' => array(
+					'color'        => 'white',
+					'height'       => '100px',
+					'border-style' => 'solid',
+				),
+			),
+		);
+
+		// Enqueues a block supports (core styles).
+		wp_style_engine_get_stylesheet_from_css_rules(
+			$core_styles_to_enqueue,
+			array(
+				'context' => 'block-supports',
+			)
+		);
+
+		$my_styles_to_enqueue = array(
+			array(
+				'selector'     => '.gandalf',
+				'declarations' => array(
+					'color'        => 'grey',
+					'height'       => '90px',
+					'border-style' => 'dotted',
+				),
+			),
+		);
+
+		// Enqueue some other styles.
+		wp_style_engine_get_stylesheet_from_css_rules(
+			$my_styles_to_enqueue,
+			array(
+				'context' => 'my-styles',
+			)
+		);
+
+		wp_enqueue_stored_styles();
+
+		$this->assertEquals(
```

Comment:
This is a hangover from my [original PR](https://github.com/WordPress/wordpress-develop/pull/3218). 

I think it's better to use `assertSame` here for string comparisons and `assertEquals` for objects etc

```suggestion
		$this->assertSame(
```